e2e/test/scenarios/onboarding/setup/setup.cy.spec.js

```diff
@@ -234,69 +234,64 @@ describe("scenarios > setup", () => {
     });
   });
 
-  ["postgres", "mysql"].forEach(engine => {
-    it(`should allow you to connect a ${engine} db during setup`, () => {
```

Comment:
@iethree please note that even if we decided to stick with these two databases, we have to always include the `@external` tag in the test config to signal that it depends on external services.

Reply:
sorry, my bad on forgetting the @external tag. I think we really wanted coverage on connecting our main transactional DBs during setup, can we do them in addition to sqlite?

Reply:
We already have E2E test that's covering that.
https://github.com/metabase/metabase/blob/master/e2e/test/scenarios/admin/databases/add-new-database.cy.spec.js

Reply:
> sorry, my bad on forgetting the @external tag.

No biggie.
Btw, that's not what broke the pre-release. It's the fact that it doesn't have QA DBs, nor `QA_DB_ENABLED` set to true.